Lib/fontmake/font_project.py

```diff
@@ -427,7 +427,8 @@ def save_otfs(
             ttf: If True, build fonts with TrueType outlines and .ttf extension.
             is_instance: If output fonts are instances, for generating paths.
             autohint: Parameters to provide to ttfautohint. If not provided, the
-                autohinting step is skipped.
+                UFO lib is scanned for autohinting parameters. If nothing is found, 
```

Comment:
maybe It's also worth mentioning the lib key name here in the --help

Reply:
The actual complete Glyphs key `com.schriftgestaltung.customParameter.InstanceDescriptorAsGSInstance.TTFAutohint options`?

Reply:
yes.. the alternative is one has to look at the code to find that out, since it's pretty obscure

Reply:
Like this?: https://github.com/googlefonts/fontmake/pull/850/commits/b23d2291803c9f77b3d07577bcf864c79f96d886

Reply:
yeah.. though the `\\` at the line end may be confusing, i'd remove that and keep on a single line (even if it may look too long or if it wraps to newline), easier to copypaste